ledger/participant-integration-api/src/main/scala/platform/apiserver/services/transaction/ApiTransactionService.scala

```diff
@@ -115,6 +125,10 @@ private[apiserver] final class ApiTransactionService private (
         )
         .via(logger.debugStream(transactionTreesLoggable))
         .via(logger.logErrorsOnStream)
+        .map(item => {
```

Comment:
I would extract a generic/extension method and instrument all sources with it for conciseness

Reply:
e.g. 
```
  implicit class InstrumentedCountingSource[E, N](s: Source[E, N]) {
    def counter(counter: Counter): Source[E, N] =
      s.map(item => {
        counter.inc()
        item
      })
  }
```
or a simple object method if extension methods don't appeal to you. What do you think?